### PR TITLE
799 subfield identifiers

### PR DESCRIPTION
**Pull request by sharpattack**
Re-opened from earlier PR.

No changes since that PR, other than this has the latest changes.

### CODE REVIEW

File: app/models/invenio_rdm_record_converter.rb
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:60]: oof, good catch!

File: app/models/invenio_rdm_record_converter.rb
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:224]: This function is fine as it is, but here's another take for you to consider. I'll leave it up to you if you'd like to change anything or not. 

```
def additional_titles(additional_titles)
  return nil if additional_titles.blank?
  additional_titles.map{ |title| {"title": title, "type": "alternative_title", "lang": ENG} }
end

additional_title(titles.shift)
```

File: app/models/invenio_rdm_record_converter.rb
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:230]: Same as the above comment. The block in that comment can be changed to fit here pretty easily. Actually, if you wanted to abstract a bit you could even write one function that handles both of these in one, but it would probably be more trouble than it's worth.

File: app/models/invenio_rdm_record_converter.rb
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:248]: haha, seems like I was the one who wrote this in the first place. I guess on a second glance I see things to change.

**Review by Meowcenary — APPROVED** [review]
Looks good to me. Some small comments, but nothing that I think necessarily needs to change.

File: spec/models/invenio_rdm_record_converter_spec.rb
Comment (fenekku) [inline spec/models/invenio_rdm_record_converter_spec.rb:378]: Are these the expected exported rights? If so, they will be changed for https://inveniordm.docs.cern.ch/reference/metadata/#rights-0-n correct?

Comment (sharpattack) [reply spec/models/invenio_rdm_record_converter_spec.rb:378]: Ah, yes. This will need to be updated once #878 is merged.

File: app/models/invenio_rdm_record_converter.rb
Comment (sharpattack) [reply app/models/invenio_rdm_record_converter.rb:248]: 😄 It happens to us all. I do like your suggested changes, first giving a shot at abstracting it.

**Comment by Meowcenary** [comment]
I just merged a PR that seems to have caused conflicts. If you have any trouble resolving these let me know.

**Review by fenekku — APPROVED** [review]

File: app/models/invenio_rdm_record_converter.rb
Comment (sharpattack) [reply app/models/invenio_rdm_record_converter.rb:247]: There were more bugs uncovered while trying to abstract. Opened a new issue, #887 to address it separately.

**Pull request merged**
